solr/core/src/java/org/apache/solr/response/transform/DocTransformers.java

```diff
@@ -88,4 +89,11 @@ public boolean needsSolrIndexSearcher() {
     return false;
   }
 
+  @Override
+  public void finish() {
+    for( DocTransformer a : children ) {
+      a.finish();
```

Comment:
what is one of them throws the exception shouldn't it continue to finish others?